assets/translations/hr-HR.json

```diff
@@ -0,0 +1,58 @@
+{
+  "Create_new_account": "Kreiraj novi račun",
+  "Actions_History": "Povijest akcija",
+  "Empty_history": "Obriši povijest",
+  "History_Settings_Page": "Stranica postavki povijesti",
+  "Lamps_Page": "Stranica za rasvjetu",
+  "Name:_": "Naziv: {}",
+  "Room:_": "Prostorija: {}",
+  "Lamps_Settings_Page": "Stranica postavki rasvjete",
+  "Welcome_to_your_Smart_Home": "Dobrodošli u svoj pametni dom",
+  "Scene_Settings_Page": "Stranica postavki prizora",
+  "Move_to_blinds_full_settings_page": "Premjesti na stranicu s punim postavkama za sjenila",
+  "Blinds_Page": "Stranica za sjenila",
+  "Down": "Dolje",
+  "Stop": "Zaustavi",
+  "Up": "Gore",
+  "Smart_Devices_Settings_Page": "Stranica postavki pametnih uređaja",
+  "Conditioner": "Uređaj za napajanje",
```

Comment:
The English `Conditioner`  is incorrect.
We are changing it to `Conditioners` same as `Air Conditioners`

Can you update to the correct new translation?

Reply:
Fixed!